orchestration/dagster_orchestration/hca_orchestration/solids/load_hca/non_file_metadata/load_non_file_metadata.py

```diff
@@ -40,6 +40,9 @@ class NonFileMetadataTypes(Enum):
     {"metadata_types": NonFileMetadataTypes, "prefix": "metadata"})
 
 
-@composite_solid
-def non_file_metadata_fanout(result: list[JobId], scratch_dataset_name: HcaScratchDatasetName) -> Nothing:
-    return ingest_non_file_metadata_type(result, scratch_dataset_name).map(load_table)
+@composite_solid(
+    output_defs=[DynamicOutputDefinition(dagster_type=Optional[JobId])]
+)
+def non_file_metadata_fanout(result: list[JobId], scratch_dataset_name: HcaScratchDatasetName) -> Optional[JobId]:
```

Comment:
Same refactoring as the `load_data_metadata_files.py` file, i.e,. removing unneeded composite -> composite structure.